src/main.js

```diff
@@ -1,33 +1,29 @@
 // Create variables targetting the relevant DOM elements here 👇
+var coverImg = document.querySelector('.cover-image');
+var coverTitle = document.querySelector('.cover-title');
+var createNew = document.querySelector('.create-new-book-button');
 var formView = document.querySelector('.form-view');
+var homeButton = document.querySelector('.home-button');
 var homeView = document.querySelector('.home-view');
+var makeNew = document.querySelector('.make-new-button');
 var randomCov = document.querySelector('.random-cover-button');
 var saveCov = document.querySelector('.save-cover-button');
```

Comment:
💯  Alphabetical order is great! We usually do this with just about everything, imports, css rules etc.